src/core/ucc_global_opts.c

```diff
@@ -30,6 +30,13 @@ ucc_config_field_t ucc_global_config_table[] = {
      ucc_offsetof(ucc_global_config_t, log_component),
      UCC_CONFIG_TYPE_LOG_COMP},
 
+    {"COLL_TRACE", "n",
+     "If UCC logging level is INFO or higher UCC will print info message"
```

Comment:
a bit confusing since you're explaining what would happen in case of "y" but defining as "n"